### PR TITLE
ollama: 0.1.39 -> 0.1.41

### PR DESCRIPTION
**Pull request by RoyDubnium**
## Description of changes
Update ollama to 0.1.41
### Changelog:

- [0.1.40](https://github.com/ollama/ollama/releases/tag/v0.1.40)
- [0.1.41](https://github.com/ollama/ollama/releases/tag/v0.1.41)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by drupol — CHANGES_REQUESTED** [review]
Hi!

Thanks for your contribution.

A couple of things to fix before merging:
1. This PR contains 2 commits, it should only contains 1
2. The commit log message is wrong, it should match the PR title

**Comment by RoyDubnium** [comment]
Thanks for your input. 
I apologise, but I am not quite sure how to fix any of these problems. The GitHub website does not appear to give me the option to amend commit log, and I don't really know where to begin. How should I proceed?

**Comment by RoyDubnium** [comment]
Nevermind, I figured it out. The problems you stated should now be fixed.

**Comment by drupol** [comment]
This PR should have 2 commits, it only has one currently.

**Comment by kirillrdy** [comment]
> This PR should have 2 commits, it only has one currently.

This contradicts your previousl comment

**Comment by drupol** [comment]
You're right, and I apologize for the confusion, I was probably distracted yesterday.

To be clear, this PR must have 2 commits. One commit to add the new maintainer, one for the rest of the changes.

**Comment by abysssol** [comment]
As far as I can tell, this pr doesn't and hasn't added a new maintaner.
@drupol Is it possible that you accidentally commented on the wrong tab or something similar?

**Comment by RoyDubnium** [comment]
To confirm.
This is my first time contributing to nixpkgs.
Does this mean I need to add myself to maintainer-list.nix?

**Comment by abysssol** [comment]
@RoyDubnium you don't have to add yourself as a maintainer unless you want to. In particular, I wouldn't recommend it unless you have any desire to contribute somewhat regularly, and sometimes be pinged in issues related to any packages you maintain, etc.

**Review by abysssol — APPROVED** [review]

File: pkgs/by-name/ol/ollama/package.nix
Comment (drupol) [inline pkgs/by-name/ol/ollama/package.nix:53]: Why this hash change? Was it a mistake from a previous commit?

**Comment by drupol** [comment]
> Ideally, I'd like to contribute to keeping ollama up to date in the future, so it might be a good idea to add myself as a maintainer, at least at some point.

Let's open a new PR for adding you as maintainer then?

**Review by drupol — APPROVED** [review]

**Comment by drupol** [comment]
Thank you @RoyDubnium for your first PR ! Feel free to create a new PR for adding you as maintainer and ping me in it when it's done.

**Comment by nonetrix** [comment]
0.1.42 fixes issue with Qwen 2 just repeating the character `g` and is fairly minor, would anyone be against bumping version to 0.1.42 so soon?

**Comment by abysssol** [comment]
@nonetrix I made a pr for 0.1.42 at #318207, I'll merge it soon

**Pull request merged**
